gradle/libs.versions.toml

```diff
@@ -54,7 +54,7 @@ secrets = "2.0.1"
 generativeai = "0.9.0"
 datastore = "1.1.4"
 lifecycleViewmodel = "1.0.0-SNAPSHOT"
-navigation3 = "0.1.0-SNAPSHOT"
+navigation3 = "1.0.0-SNAPSHOT"
```

Comment:
I think we should have a new version number for `adaptive-navigation3`, I don't think there's any guarantee it will match the `navigation3` version number.